app/src/main/java/org/oppia/android/app/help/HelpActivityPresenter.kt

```diff
@@ -16,11 +16,13 @@ class HelpActivityPresenter @Inject constructor(private val activity: AppCompatA
   private lateinit var toolbar: Toolbar
 
   fun handleOnCreate(isFromNavigationDrawer: Boolean) {
-    activity.setContentView(R.layout.help_activity)
-    setUpToolbar()
     if (isFromNavigationDrawer) {
+      activity.setContentView(R.layout.help_activity)
+      setUpToolbar()
       setUpNavigationDrawer()
     } else {
+      activity.setContentView(R.layout.help_without_drawer_activity)
+      setUpToolbar()
```

Comment:
`setUpToolbar` function is comment and therefore can be pulled out.

Reply:
@rt4914 
we cannot pull it up because without  `setContentView` we cannot call `setupToolbar` as this function uses the view attribute.
without setting the content view we cannot call it.

and also we cannot pull It down because the following up functions cannot be called before `setuptollbar` function.

Reply:
Yes I understand that but can't we take this outside but below the `if-else` condition?

Reply:
@rt4914 
no, we cannot take it outside as `setupNaviagtionDrawer`  ( line 22 ) cannot be called before `setupToolbar`.

Reply:
Ack.